sheet.go

```diff
@@ -78,10 +78,17 @@ func (s *Sheet) Col(idx int) *Col {
 // containing the data from the field "A1" on the spreadsheet.
 func (sh *Sheet) Cell(row, col int) *Cell {
 
-	if len(sh.Rows) > row && sh.Rows[row] != nil && len(sh.Rows[row].Cells) > col {
-		return sh.Rows[row].Cells[col]
+	// If the user requests a row beyond what we have, then extend.
+	for len(sh.Rows) <= row {
```

Comment:
Since xlsx files don't contain data for non-existent rows or columns, it seems like this would unnecessarily bloat the resulting file with potentially hundreds of thousands of empty rows. Have you manually tested this using, for example, row 100,000?